src/Resources/app/administration/src/module/sw-cms/elements/nosto-integration/component/sw-cms-el-nosto-integration.scss

```diff
@@ -1,14 +1,15 @@
-.sw-cms-block-nosto-integration-block {
```

Comment:
The administration changes are part of the automatic code review from Shopware. During that, I noticed some issues with the styling